ui/package.json

```diff
@@ -7,6 +7,7 @@
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.58",
     "@material-ui/styles": "^4.11.4",
+    "@reduxjs/toolkit": "^1.6.0",
```

Comment:
This library has 10MB (unpacked size), with 4 dependencies. Do we really need this to create a simple reducer?

Reply:
Sorry for the late reply; while going through redux docs, it was mentioned that `reduxjs/toolkit` is easily maintainable, so I thought if someone has to make changes in the future, it will be easy for them to understand. Should I remove it?